ee/tables/homebrew/upgradeable.go

```diff
@@ -43,7 +44,12 @@ func (t *Table) generate(ctx context.Context, queryContext table.QueryContext) (
 	// that user. To reduce duplicating the WithUid table helper, we can find the owner of the binary,
 	// and pass the said owner to the WIthUid method to handle setting the appropriate env vars.
 	cmd, err := allowedcmd.Brew(ctx)
+		
```

Comment:
Apologies, missed this previously -- looks like the lint job is complaining about this line as well. (If you want to validate for yourself before CI, you can [install golangci-lint](https://golangci-lint.run/welcome/install/#local-installation) and then run `golangci-lint run`.)